Index: core/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- core/build.gradle	(revision 37a85aa005d0945ab3642a71f8d9f6e6ca44f307)
+++ core/build.gradle	(revision )
@@ -36,6 +36,13 @@
     PUBLISH_VERSION = clientModuleVersion
 }
 
+test {
+    reports {
+        junitXml.enabled = true
+        html.enabled = false
+    }
+}
+
 dependencies {
     compile 'com.intellij:annotations:12.0'
     compile 'com.squareup.okhttp:okhttp:2.5.0'
\ No newline at end of file
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- build.gradle	(revision 37a85aa005d0945ab3642a71f8d9f6e6ca44f307)
+++ build.gradle	(revision )
@@ -42,19 +42,4 @@
         jcenter()
         maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
     }
-}
+}
\ No newline at end of file
-
-
-// ensure clean is also triggered for root build folder
-apply plugin: 'java'
-apply plugin: 'build-dashboard'
-
-buildDashboard {
-    reports.html.destination = "build/"
-}
-
-// just clean up dashboard from not generated reports
-test.reports.html.enabled = false
-// just clean up dashboard from not generated reports
-test.reports.junitXml.enabled = false
-
\ No newline at end of file
Index: core-integration-tests/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- core-integration-tests/build.gradle	(revision 37a85aa005d0945ab3642a71f8d9f6e6ca44f307)
+++ core-integration-tests/build.gradle	(revision )
@@ -51,6 +51,13 @@
     testServersName = "test-servers.json"
 }
 
+test {
+    reports {
+        junitXml.enabled = true
+        html.enabled = false
+    }
+}
+
 task prepIntegrationEnv() {
     def buildDir = file(envPath)
     requiredProperties "servers.file.name", "run.resource.count"
\ No newline at end of file
Index: rx/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- rx/build.gradle	(revision 37a85aa005d0945ab3642a71f8d9f6e6ca44f307)
+++ rx/build.gradle	(revision )
@@ -36,6 +36,13 @@
     PUBLISH_VERSION = clientModuleVersion
 }
 
+test {
+    reports {
+        junitXml.enabled = true
+        html.enabled = false
+    }
+}
+
 dependencies {
     compile project(':js-android-sdk-core')
     compile 'io.reactivex:rxjava:1.1.0'
\ No newline at end of file
